### DISCUSSION TITLE
Add support for Android 12 Material You UI colors

### DISCUSSION THREAD

**Discussion by ShivamJoker** [react-native-community/discussions-and-proposals#445] [Ideas]
Android 12 introduces dynamic colors which can be easily accessed via APIs 

There is a library which does this but it would be better to have it in React Native itself, like platform color.

https://github.com/Assembless/react-native-material-you

**Comment by p-syche** [comment]
Hi @ShivamJoker ,
Since this is a functionality specific for Android only I don't agree it should be included in React Native. If any dev wants to take advantage of this specific functionality they can add a specialized lib, like the one you linked.

**Reply by ShivamJoker** [reply]
There are many features which are platform specific but still available in React Native like Dynamic Color iOS and Toast Android 🙄

**Reply by p-syche** [reply]
That's true, they are available. But I think they shouldn't be :) That's just my point of view.
